docs/fsharp/language-reference/members/explicit-fields-the-val-keyword.md

```diff
@@ -53,7 +53,7 @@ The following code shows the use of explicit fields in a class that does not hav
 
 The output is `35 22`.
 
-The following code shows the use of explicit fields in a structure. Because a structure is a value type, it automatically has a default constructor that sets the values of its fields to zero. Therefore, the `DefaultValue` attribute is not required.
+The following code shows the use of explicit fields in a structure. Because a structure is a value type, it automatically has a parameterless constructor that sets the values of its fields to zero. Therefore, the `DefaultValue` attribute is not required.
```

Comment:
please revert

Reply:
I've just removed this change by amending this commit